recipe/meta.yaml

```diff
@@ -1,21 +1,19 @@
-{% set version = "4.2.2" %}
+{% set version = "5.2.0" %}
 
 package:
   name: bazel
   version: {{ version }}
 
 source:
   url: https://github.com/bazelbuild/bazel/releases/download/{{ version }}/bazel-{{ version }}-dist.zip
-  sha256: 9981d0d53a356c4e87962847750a97c9e8054e460854748006c80f0d7e2b2d33
+  sha256: 820a94dbb14071ed6d8c266cf0c080ecb265a5eea65307579489c4662c2d582a
   patches:
     - patches/0001-allow-args-to-be-passed-to-bazel_build.patch  # [unix]
     - patches/0002-do-not-use-enable-gold-unless-supported.patch  # [unix]
-    - patches/0003-Use-LSFindApplicationForInfo-for-locating-xcode.patch  # [osx]
```

Comment:
Asking the obvious, but I guess these were removed because of not being needed anymore.

Reply:
I don't think this patch was ever strictly needed and it doesn't work anymore.